tests/test_norm.py

```diff
@@ -72,10 +74,21 @@ def test_cumnorm_serial():
         threads_per_worker=1,
         random_seed=100
     )
-    ref_data = torch.cat([batch.x for batch in ds], dim=0)
 
+    return ds
+
+
+@pytest.fixture(scope='session')
+def serial_ref_data(train_dataset: EdgeDataset) -> torch.Tensor:
```

Comment:
I am learning the use of `pytest.fixture(scope='session')` here :) Since you use the `session`, this function is called and being used in all other small tests below 👍

Reply:
That's correct. In this way, we don't have to re-run anything that is a fixture because it becomes available to each function as an input.